lib/rbac/filterer.rb

```diff
@@ -358,6 +358,26 @@ def select_from_order_columns(columns)
       end
     end
 
+    # Given a nested hash of associations (used by includes)
+    #   convert into an array of table names (used by references)
+    # If given an array of table names, will output the same array
+    def self.includes_to_references(klass, inc)
```

Comment:
is this a function of rbac?  Rbac filtering? Or generic enough to be a AR mixin?

Reply:
ar mixin